api/index.js

```diff
@@ -72,32 +70,12 @@ app.get('/comments', async (req, res) => {
     return msg.value.content.tangles.comment.root === null // comparing to see if it's equal, if it is, then true
   }) // filter works on arrays, only keeps some elements if it returns true you want to keep it if it returns false, you don't want to keep the messages of the array
 
-  comments = await Promise.all(comments.map(msg => commentCrut.read(msg.key))) // with each of these messages, use crut to load the comments, please!
-  res.send({ comments }) // only gives back the comment text, could also show author, which is automatically added to the messages in ssb
-})
-
-
-
-// THIS ONE IS THE BASE FOR GETTING 10 LATEST COMMENTS (using crut)
-app.get('/10LatestCommentsCrut', async (req, res) => {
-  const { where, and, type, toPromise } = require('ssb-db2/operators') // enables the commands to be used
-  let comments = await ssb.db.query( // used to be "const" instead of "let"
-    where( // select inside the list with the requirements of multiple things, "and"
-      and( // two things need to be true
-        type('comment') // specifically comments
-      )
-    ), toPromise() // whenever it has promised the await is waiting the responses
-  )
-  comments = comments.filter(msg => {
-    return msg.value.content.tangles.comment.root === null // comparing to see if it's equal, if it is, then true
-  }) // filter works on arrays, only keeps some elements if it returns true you want to keep it if it returns false, you don't want to keep the messages of the array
```

Comment:
We were able to delete this whole method by folding it together with `/comments`

you use it like `http://localhost:3000/comments?limit=10`